plugins/topology/src/components/Topology/TopologySideBar/TopologyVirtualMachineDetails.tsx

```diff
@@ -0,0 +1,114 @@
+  const devices = getLabeledDevices(vm) || [];
+  const ipAddrs = getVmiIpAddresses(vmi); // Need data to test
+  const description = getDescription(vm);
+  const templateName = getLabel(vm, LABEL_USED_TEMPLATE_NAME);
+  const nodeName = getVMINodeName(vmi) || getNodeName(pods);
+  // const vmiReady = isVMIReady(vmi);
+  // const [sshService] = useSSHService2(vmi);
+
+  return (
+    <>
+      <div className="topology-workload-details">
+        <TopologyWorkloadDetails resource={vm}>
+          <TopologySideBarDetailsItem label="Status">
+            {vm?.status?.printableStatus}
+          </TopologySideBarDetailsItem>
+        </TopologyWorkloadDetails>
+      </div>
+      <div
+        className="topology-workload-details"
+        data-testid="deployment-details"
+      >
+        <TopologySideBarDetailsItem
+          label="Description"
+          emptyText="Not available"
+        >
+          {description}
+        </TopologySideBarDetailsItem>
+        <TopologySideBarDetailsItem label="Operating System">
+          Need to add it
+        </TopologySideBarDetailsItem>
+        <TopologySideBarDetailsItem label="Template">
+          <ResourceIcon kind="Template" />
+          {templateName}
+        </TopologySideBarDetailsItem>
+        <TopologySideBarDetailsItem label="Pod">
+          {pods?.map(pod => <label>{pod?.metadata?.name}</label>)}
+        </TopologySideBarDetailsItem>
+        <TopologySideBarDetailsItem label="Boot order">
+          <BootOrderSummary devices={devices} />
+        </TopologySideBarDetailsItem>
+        <TopologySideBarDetailsItem label="IP address">
+          {ipAddrs}
+        </TopologySideBarDetailsItem>
+        <TopologySideBarDetailsItem label="Hostname">
+          {vm.spec?.strategy?.type}
+        </TopologySideBarDetailsItem>
+        <TopologySideBarDetailsItem label="Time zone">
+          {vm.spec?.strategy?.type}
+        </TopologySideBarDetailsItem>
+        <TopologySideBarDetailsItem label="Node" emptyText="Not available">
+          {nodeName}
+        </TopologySideBarDetailsItem>
+        <TopologySideBarDetailsItem label="Workload profile">
+          {vm.spec?.strategy?.type}
+        </TopologySideBarDetailsItem>
+        <TopologySideBarDetailsItem label="User credentials">
+          {vm.spec?.strategy?.type}
+        </TopologySideBarDetailsItem>
+        <TopologySideBarDetailsItem label="SSH access">
+          {/* <span data-test="details-item-ssh-access-port">
+          {vmiReady ? 
+            sshServicesRunning ?  `Port: ${sshServicePort}` 
+             : 'SSH service disabled'
+            
+          : 
+            'Virtual machine not running')
+          }
+        </span> */}
+        </TopologySideBarDetailsItem>
+        <TopologySideBarDetailsItem label="Hardware devices">
+          <div>GPU devices : {getGPUDevices(vm).length}</div>
+          <div>Host devices : {getHostDevices(vm).length}</div>
+        </TopologySideBarDetailsItem>
```

Comment:
Please remove the spaces after the "keys"

```suggestion
        <TopologySideBarDetailsItem label="Hardware devices">
          <div>GPU devices: {getGPUDevices(vm).length}</div>
          <div>Host devices: {getHostDevices(vm).length}</div>
        </TopologySideBarDetailsItem>
```

Reply:
Done